src/components/Modal/components/ModalWrapper/ModalWrapper.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import { withTheme } from 'emotion-theming';
+
+import Card from '../../../Card';
+
+const cardStyles = ({ theme }) => css`
```

Comment:
I guess, in this case, we could rename this to `baseStyles` for consistency.

Reply:
Changed, thanks.